common/tier4_state_rviz_plugin/src/autoware_state_panel.cpp

```diff
@@ -123,120 +68,168 @@ AutowareStatePanel::AutowareStatePanel(QWidget * parent) : rviz_common::Panel(pa
   setLayout(v_layout);
 }
 
+QGroupBox * AutowareStatePanel::makeOperationModeGroup()
+{
+  auto * group = new QGroupBox("OperationMode");
+  auto * grid = new QGridLayout;
+
+  operation_mode_label_ptr_ = new QLabel("INIT");
+  operation_mode_label_ptr_->setAlignment(Qt::AlignCenter);
+  operation_mode_label_ptr_->setStyleSheet("border:1px solid black;");
+  grid->addWidget(operation_mode_label_ptr_, 0, 0, 0, 1);
+
+  auto_button_ptr_ = new QPushButton("AUTO");
+  auto_button_ptr_->setCheckable(true);
+  connect(auto_button_ptr_, SIGNAL(clicked()), SLOT(onClickAutonomous()));
+  grid->addWidget(auto_button_ptr_, 0, 1);
+
+  stop_button_ptr_ = new QPushButton("STOP");
+  stop_button_ptr_->setCheckable(true);
+  connect(stop_button_ptr_, SIGNAL(clicked()), SLOT(onClickStop()));
+  grid->addWidget(stop_button_ptr_, 0, 2);
+
+  local_button_ptr_ = new QPushButton("LOCAL");
+  local_button_ptr_->setCheckable(true);
+  connect(local_button_ptr_, SIGNAL(clicked()), SLOT(onClickLocal()));
+  grid->addWidget(local_button_ptr_, 1, 1);
+
+  remote_button_ptr_ = new QPushButton("REMOTE");
+  remote_button_ptr_->setCheckable(true);
+  connect(remote_button_ptr_, SIGNAL(clicked()), SLOT(onClickRemote()));
+  grid->addWidget(remote_button_ptr_, 1, 2);
+
+  group->setLayout(grid);
+  return group;
+}
+
+QGroupBox * AutowareStatePanel::makeControlModeGroup()
+{
+  auto * group = new QGroupBox("ControlMode");
+  auto * grid = new QGridLayout;
+
+  control_mode_label_ptr_ = new QLabel("INIT");
+  control_mode_label_ptr_->setAlignment(Qt::AlignCenter);
+  control_mode_label_ptr_->setStyleSheet("border:1px solid black;");
+  grid->addWidget(control_mode_label_ptr_, 0, 0);
+
+  autoware_control_button_ptr_ = new QPushButton("Autoware");
+  autoware_control_button_ptr_->setCheckable(true);
+  connect(autoware_control_button_ptr_, SIGNAL(clicked()), SLOT(onClickAutowareControl()));
+  grid->addWidget(autoware_control_button_ptr_, 0, 1);
+
+  direct_control_button_ptr_ = new QPushButton("Direct");
```

Comment:
control mode has autonomous manual no command so I think this "direct" should be "manual" which is same as message type
https://github.com/tier4/autoware_auto_msgs/blob/bd162589a837a2c1ecea943f80c035109e1ae594/autoware_auto_vehicle_msgs/msg/ControlModeCommand.idl#L8

Reply:
@taikitanaka3 @h-ohta 
More precisely, it indicates a mode in which the vehicle is not controlled by Autoware. There is no guarantee that this is a manual operation, for example, if the vehicle itself has ADAS, line trace, and so on. So, it is expressed as direct control in the API documentation.